core/src/main/java/org/jobrunr/scheduling/JobScheduler.java

```diff
@@ -292,6 +294,10 @@ public String scheduleRecurrently(JobLambda job, String cron) {
         return scheduleRecurrently(null, job, cron);
     }
 
+    public String scheduleRecurrently(String cron, Function<?> lambda) {
```

Comment:
Yep! Thanks for clarifying. In Kotlin, if a lambda is the last argument of a function, it can be moved outside of the parameters, eg.:
```kotlin
jobScheduler.scheduleRecurrently("0 1 * * 1-5") {
  doSomething()
}
```
as opposed to:
```kotlin
jobScheduler.scheduleRecurrently({   doSomething() }"0 1 * * 1-5")
```
For that reason, it's a common Kotlin idiom to have a lambda as the last parameter of a function call